docs/src/css/components/_toast.scss

```
@@ -135,17 +135,16 @@ ix-toast {
   }
 
-    > aside[data-intent="gradient"] {
-
-    border-left: 3px solid var(--ix-gradient);
+    > aside[data-intent='gradient'] {
+    border: 1px solid #ffffff;
+    border-left: 3px solid #ffffff;
  
-    background: var(--ix-gradient-bg);
-
+     background: var(--ix-gradient-bg);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)